eng/Versions.props

```diff
@@ -1,6 +1,6 @@
 <Project>
   <PropertyGroup>
     <VersionPrefix>3.1.0</VersionPrefix>
-    <PreReleaseVersionLabel>preview3</PreReleaseVersionLabel>
+    <PreReleaseVersionLabel>preview4</PreReleaseVersionLabel>
```

Comment:
@mkArtakMSFT Just double checking. I know you said this branding update to preview4 should go in the blazor-wasm branch. Is the same true for master?

Reply:
Yes, true for `master`!

Reply:
I didn't realize this at first, GitHub is being a little weird here with the diff because there are conflicts. Here's the current version of Versions.props in master:

https://github.com/aspnet/Blazor/blob/075a3dd8e393750c815ee2a1e27a5dff51c6e017/eng/Versions.props#L1-L6

As you can see, the current version in master is 5.0.0-alpha1. I assume you want to keep it 5.0.0-alpha1, right?

Reply:
Yes, that's fine for the master branch. At least for now: we do have time to tweak this in case we need to